ivy/functional/backends/torch/statistical.py

```diff
@@ -106,9 +110,9 @@ def prod(
                     ]
                 ),
                 dtype=dtype,
-                out=out,
+                **out
             )
-    return torch.prod(input=x, dim=axis, dtype=dtype, keepdim=keepdims, out=out)
+    return torch.prod(input=x, dim=axis, dtype=dtype, keepdim=keepdims, **out)
```

Comment:
does passing `out=None` fail here, otherwise, what is the point of this `out` dict?